lib/sles4sap/console_redirection.pm

```diff
@@ -43,33 +43,133 @@ sub handle_login_prompt {
     my $pwd = get_var('_SECRET_SUT_PASSWORD', $testapi::password);
     set_serial_term_prompt();
     # look for either password prompt or command prompt to appear
-    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20);
+    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20, quiet => 1);
 
     die 'Neither password not command prompt appeared.' unless $serial_response;
     # Handle password prompt if it appears
     if (grep /Password:\s*$/, $serial_response) {
         type_password $pwd;
         send_key 'ret';
         # wait for command prompt to be ready
-        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20);
+        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20, quiet => 1);
     }
     set_serial_term_prompt();    # set correct serial prompt
 }
 
 =head2 redirection_init
 
-    redirection_init();
+    redirection_init( [, ssh_user=>$ssh_user, target_ip=>$target_ip, ssh_tunnel_port=>$ssh_tunnel_port]);
+
+B<ssh_user>: Target login user - default value is defined by OpenQA parameter REDIRECT_TARGET_USER
+
+B<target_ip>: Target host IP - default value is defined by OpenQA parameter REDIRECT_TARGET_IP
+
+B<ssh_tunnel_port>: Local port used for reverse ssh tunnel, default: 22022
+
+Does initial setup for console redirection which includes:
+
+=over
+
+=item * sets BASE_VM_ID value with current machine id. This helps identifying the base (not redirected) VM.
 
-Do preparation before redirecting console. Gets base VM id and initial setup.
-This is required to be done only once at the beginning of the whole test.
-If you have a multi machine setup, execute this on each worker VM.
+=item * SSH key exchange for reverse ssh connection.
 
+=item * remote port forwarding for reverse SSH session
+
+=item * remote port forwarding of incoming traffic from remote VM to OpenQA server resources (upload_logs, download from 'data')
```

Comment:
`remote VM`

Reply:
Changed to: 
```
=item * remote port forwarding of incoming traffic from B<destination_ip> to OpenQA
    server resources (upload_logs, download from 'data')
```
using 'destination' is a bit of a problem here since destination for traffic forwarding is  OpenQA server not the cloud VM. On the other side Cloud VM is destination for console redirection.